near-plugins/src/access_control_role.rs

```diff
@@ -0,0 +1,10 @@
+pub trait AccessControlRole {
+    /// Returns the bitflag corresponding to the super admin permission.
+    fn acl_super_admin_permission_bitflag() -> u128;
+
+    /// Returns the bitflag corresponding to the admin permission for the role.
+    fn acl_admin_permission_bitflag(self) -> u128;
+
+    /// Returns the bitflag corresponding to the role's permission.
+    fn acl_permission_bitflag(self) -> u128;
```

Comment:
What do you think about removing the suffix `_bitflag`

Tbh, I don't expect user to be using these functions directly most of the time, so doesn't really matter much.

Reply:
True with `_bitflag` it's quite verbose. Removed in 2793779.